tests/integration/s3/test_s3.py

```diff
@@ -4752,6 +4752,38 @@ def test_s3_intelligent_tier_config(self, aws_client, s3_create_bucket, snapshot
         response = aws_client.s3.list_bucket_intelligent_tiering_configurations(Bucket=bucket)
         snapshot.match("list_bucket_intelligent_tiering_configurations_3", response)
 
+    @pytest.mark.aws_validated
+    def test_s3_get_object_headers(self, aws_client, s3_create_bucket, snapshot):
+        bucket = s3_create_bucket()
+        key = "en-gb.wav"
+        file_path = os.path.join(f"../files/{key}")
```

Comment:
nit: we can use this so that the test is runnable from anywhere
```suggestion
        file_path = os.path.join(
            os.path.dirname(__file__),
            f"../files/{key}",
        )
```